### PR TITLE
fix wrong order of require.Equal arguments

### PR DESCRIPTION
**Pull request by tadelesh**
- [ ] The purpose of this PR is explained in this or a referenced issue.
- [ ] The PR does not update generated files.
   - These files are managed by the codegen framework at [Azure/autorest.go][].
- [ ] Tests are included and/or updated for code changes.
- [ ] Updates to [CHANGELOG.md][] are included.
- [ ] MIT license headers are included in each file.

[Azure/autorest.go]: https://github.com/Azure/autorest.go
[CHANGELOG.md]: https://github.com/Azure/azure-sdk-for-go/blob/main/CHANGELOG.md

### CODE REVIEW

File: sdk/containers/azcontainerregistry/client_test.go [hunk 1]
```diff
@@ -342,13 +342,13 @@ func TestClient_UpdateManifestProperties(t *testing.T) {
 	},
 	})
 	require.NoError(t, err)
-	require.Equal(t, *res.Manifest.ChangeableAttributes.CanWrite, false)
+	require.Equal(t, false, *res.Manifest.ChangeableAttributes.CanWrite)
```
Comment (chlowell) [inline sdk/containers/azcontainerregistry/client_test.go:345]:
```suggestion
	require.False(t, *res.Manifest.ChangeableAttributes.CanWrite)
```

File: sdk/containers/azcontainerregistry/client_test.go [hunk 2]
```diff
@@ -342,13 +342,13 @@ func TestClient_UpdateManifestProperties(t *testing.T) {
...
 	res, err = client.UpdateManifestProperties(ctx, "alpine", digest, &ClientUpdateManifestPropertiesOptions{Value: &ManifestWriteableProperties{
 		CanWrite: to.Ptr(true),
 	},
 	})
 	require.NoError(t, err)
-	require.Equal(t, *res.Manifest.ChangeableAttributes.CanWrite, true)
+	require.Equal(t, true, *res.Manifest.ChangeableAttributes.CanWrite)
```
Comment (chlowell) [inline sdk/containers/azcontainerregistry/client_test.go:351]:
```suggestion
	require.True(t, *res.Manifest.ChangeableAttributes.CanWrite)
```

File: sdk/containers/azcontainerregistry/client_test.go [hunk 3]
```diff
@@ -372,13 +372,13 @@ func TestClient_UpdateRepositoryProperties(t *testing.T) {
 	},
 	})
 	require.NoError(t, err)
-	require.Equal(t, *res.ContainerRepositoryProperties.ChangeableAttributes.CanWrite, false)
+	require.Equal(t, false, *res.ContainerRepositoryProperties.ChangeableAttributes.CanWrite)
```
Comment (chlowell) [inline sdk/containers/azcontainerregistry/client_test.go:375]:
```suggestion
	require.False(t, *res.ContainerRepositoryProperties.ChangeableAttributes.CanWrite)
```

File: sdk/containers/azcontainerregistry/client_test.go [hunk 4]
```diff
@@ -372,13 +372,13 @@ func TestClient_UpdateRepositoryProperties(t *testing.T) {
...
 	res, err = client.UpdateRepositoryProperties(ctx, "ubuntu", &ClientUpdateRepositoryPropertiesOptions{Value: &RepositoryWriteableProperties{
 		CanWrite: to.Ptr(true),
 	},
 	})
 	require.NoError(t, err)
-	require.Equal(t, *res.ContainerRepositoryProperties.ChangeableAttributes.CanWrite, true)
+	require.Equal(t, true, *res.ContainerRepositoryProperties.ChangeableAttributes.CanWrite)
```
Comment (chlowell) [inline sdk/containers/azcontainerregistry/client_test.go:381]:
```suggestion
	require.True(t, *res.ContainerRepositoryProperties.ChangeableAttributes.CanWrite)
```

File: sdk/containers/azcontainerregistry/client_test.go [hunk 5]
```diff
@@ -405,13 +405,13 @@ func TestClient_UpdateTagProperties(t *testing.T) {
 	},
 	})
 	require.NoError(t, err)
-	require.Equal(t, *res.Tag.ChangeableAttributes.CanWrite, false)
+	require.Equal(t, false, *res.Tag.ChangeableAttributes.CanWrite)
```
Comment (chlowell) [inline sdk/containers/azcontainerregistry/client_test.go:408]:
```suggestion
	require.False(t, *res.Tag.ChangeableAttributes.CanWrite)
```

File: sdk/containers/azcontainerregistry/client_test.go [hunk 6]
```diff
@@ -405,13 +405,13 @@ func TestClient_UpdateTagProperties(t *testing.T) {
...
 	res, err = client.UpdateTagProperties(ctx, "alpine", "3.17.1", &ClientUpdateTagPropertiesOptions{Value: &TagWriteableProperties{
 		CanWrite: to.Ptr(true),
 	},
 	})
 	require.NoError(t, err)
-	require.Equal(t, *res.Tag.ChangeableAttributes.CanWrite, true)
+	require.Equal(t, true, *res.Tag.ChangeableAttributes.CanWrite)
```
Comment (chlowell) [inline sdk/containers/azcontainerregistry/client_test.go:414]:
```suggestion
	require.True(t, *res.Tag.ChangeableAttributes.CanWrite)
```

File: sdk/resourcemanager/internal/testutil/helper_test.go [hunk 7]
```diff
@@ -36,7 +36,7 @@ func TestCreateDeleteResourceGroup(t *testing.T) {
 	defer stop()
 	resourceGroup, _, err := CreateResourceGroup(ctx, subscriptionID, cred, options, "eastus")
 	require.NoError(t, err)
-	require.Equal(t, strings.HasPrefix(*resourceGroup.Name, "go-sdk-test-"), true)
+	require.Equal(t, true, strings.HasPrefix(*resourceGroup.Name, "go-sdk-test-"))
```
Comment (chlowell) [inline sdk/resourcemanager/internal/testutil/helper_test.go:39]:
```suggestion
	require.True(t, strings.HasPrefix(*resourceGroup.Name, "go-sdk-test-"))
```

**Review by chlowell — APPROVED** [review]

**Review by RickWinter — APPROVED** [review]
Agree with using require.True and require.False in the places Charles mentioned.

**Pull request merged**